vignettes/standardize_parameters_effsize.Rmd

```diff
@@ -293,6 +294,15 @@ the related terms, so that they correspond to changes of 1 SD of the predictor
 are still related to changes in levels. This method is not accurate and tend to
 give aberrant results when interactions are specified.
 
+But why is interaction for posthoc-standardization problematic? A regression model
+estimates coefficient between two variables when the other predictors are at 0
+(are *fixed* at 0, that people interpret as *"adjusted for"*). When a standardized
+data is passed (in the *refit* method), the effects and interactions are estimated
+at the *means* of the other predictors (because 0 is the mean for a standardized
+variable). Whereas in posthoc-standardization, this coefficient corresponds to
+something different (because the 0 has different meanings in standardized and
+non-standardized data).
+
```

Comment:
@mattansb in the last commit, I added this paragraph, because I find it quite helpful.